twd.htm

```diff
@@ -215249,30 +215166,30 @@ <h3><a id="Year1997">1997</a></h3>
 1x Mitchell, The Headhunter	 2   obt pot		  	    Pander:2
 
 Library Cards
-6	Action 		Govern the Unaligned		Dominate
-1	Action 		Bum's Rush
-7	Action Modifier Bonding				Dominate
-6	Action Modifier Conditioning			Dominate
-8	Action Modifier Shadow Play			Obtenebration
-3	Action Modifier Shroud of Night			Obtenebration
-3	Combat		Arms of the Abyss		Obtenebration
-3	Combat		Entombment			Obtenebration
-5	Combat		Shadow Body			Obtenebration
-4	Equipment	Leather Jacket
-1 	Master		Archon Investigation
-4	Master		Blood Doll
-2	Master		Minion Tap
-3	Master		Dominate
-2	Master		Obtenebration
-3	Master		Elysian Fields			Lasombra
-2	Master		Elysium: The Arboretum
-3	Master		Political Hunting Ground	Lasombra
-2	Master 		Slave Auction
-1 	Master		Sudden Reversal
-5	Reaction	Deflections			Dominate
-5	Reaction	Eyes of the Night		Obtenebration
-6	Reaction	Redirection			Dominate
-6	Reaction	Wake with Evening's Freshness
+6	Govern the Unaligned
+1	Bum's Rush
+7	Bonding
+6	Conditioning
+8	Shadow Play
+3	Shroud of Night
+3	Arms of the Abyss
+3	Entombment
+5	Shadow Body
+4	Leather Jacket
+1 Archon Investigation
```

Comment:
Why the variance in spacing?

Reply:
tabs versus spaces, won't matter to the parser